src/lib/installer/index.js

```diff
@@ -1,6 +1,8 @@
 var ghdownload = require('download-github-repo')
```

Comment:
Could we change these `require` to es6 `import` statements and the module.exports to `export`? I know we have other lib modules that have still these `require`s. We may take an approach of converting an old module as we need a change there.